### ISSUE TITLE
$el of undefined in tbody

### ISSUE THREAD

**Issue by alirezahi** [bootstrap-vue/bootstrap-vue#4417]
### Describe the bug
When I put a `b-button` inside table and click on it, it will raise an error.  Which is this:
```
TypeError: Cannot read property '$el' of undefined.
in getTbody (mixin-tbody.js?399f:32) // line 32
```

### Steps to reproduce the bug

1.Put a `b-table` element
2.Make a cell (using template slot) which has a `b-button` in it
3.When you click on the button you will see the error.

### Expected behavior

From b-button to act as a normal button

### Versions

**Libraries:**

- BootstrapVue: 2.1.0
- Bootstrap: 4.#.#
- Vue: 2.#.#

And I know it is ok in version `2.0.4`

**Environment:**

- Device: [PC]
- OS: [Ubuntu]
- Browser: [Chrome]
- Version: [70]

**Comment by tmorehouse** [comment]
Duplicate issue regarding the row-hovered event. PR #4388 fixes this and will be available in v2.2.0 coming out soon.

**Comment by tmorehouse** [comment]
Fix available in BootstrapVue v2.2.0

**Comment by DAIMER2001** [comment]
I have the same problem, I'm from Colombia, I don't speak English, I hope I can find the solution
